### PR TITLE
sign up view and styles corrections

### PR DESCRIPTION
**Pull request by cleysonsilvame**

### CODE REVIEW

**Review by LuigiRamires — COMMENTED** [review]
Very good, continue doing that **amazing** work! ♻️

File: public/css/styles.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
```
Comment (MatheusFranciscone) [inline public/css/styles.css:1]: Nice man! 😄

**Review by MatheusFranciscone — APPROVED** [review]
👍

**Review by cleysonsilvame — COMMENTED** [review]
assinatura-cleyson-silva

- ### [Acesse meu GitHub](github.com/cleysonsilvame)
- ### [Acesse meu Instragram](instagram.com/cleysonsilva.me)
- ### [Acesse meu Linkedin](linkedin.com/in/cleyson-silva-639b01188)

**Pull request merged**